<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardMenu</class>
 <widget class="QWidget" name="BoardMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="navigation_group">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Player Navigation</string>
   </property>
   <widget class="QPushButton" name="up_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="misc_group">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Misc.</string>
   </property>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset </string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="dice_field">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dice_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Moves Left for Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="roll_die_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll Die</string>
    </property>
   </widget>
   <widget class="QLabel" name="dice_label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Player</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="current_player_field">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="player_order_group_box">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Player Order</string>
   </property>
   <widget class="QLabel" name="player_one_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player One</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="player_one_pos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="player_two_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Two</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="player_two_pos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="player_three_pos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="player_four_pos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="player_three_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Three</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_four_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player Four</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
